content/outcomes/secure-software-supply-chain/establish-capability/establish-devsecops.md

```diff
@@ -0,0 +1,108 @@
+means that these teams are not working collaboratively towards a common goal.
+
+Because these teams each have different objectives, friction often develops when
+attempting to achieve outcomes. Development teams are often blindsided by
+unclear or poorly understood security requirements, operations teams do not
+usually have nuanced understanding of the code to effectively operate it, and
+security teams struggle to keep the business secure in the face of rapidly
+evolving technical requirements.
+
+While Secure Software Supply Chains are primarily a component of getting code
+into production, they are also an opportunity to reduce the friction between the
+various stakeholders attached to software development efforts. When implemented
+properly, each part of the organization will be working together towards the
+common goal of delivering customer value through software delivery.
+
+## Changing Responsibilities
+
+Members of a development organization will still maintain their respective areas
+of expertise, but they will now be asked to work more collaboratively towards
+this common goal of bringing code to production. Instead of serialized tasks,
+within DevSecOps each of these groups are working together in parallel. Needs
+and solutions are clearly communicated on a regular basis. And, because these
+lines of reponsibility are becoming blurred, members can come to shared
+understandings of how the system works through code. After all, they are now
+working as one team.
+
+### The Platform Operator
+
+The role of the SRE or platform operator shifts from one of being primarily
+focused on system uptime to one where the operator is an integral part of the
+development process itself. While the application teams will be focused on new
+feature development, SREs will focus on the reliability of the whole system.
+This will require upleveled skills from that of a classic system administrator
+role. Instead of a simple restart of a service, an SRE will now be analyzing
+performance metrics, debugging defects, and helping to build plans to ensure
+data integrity.
+
+Maintaining the supply chain will require the development of cross-cutting
+functionality: pipelines and tools that are suitable for all development teams.
+This may include implementation of common frameworks for business continuity,
+systems that enforce transparent security, and general infrastructure
+management.
+
+### The Security Engineer
+
+The role of the security engineer, similar to that of a platform engineer,
+evolves into one that is closely aligned with their development partners.
+Instead of acting as a gate to production, they are now engaged with the
+development teams from the start. The goal is to ensure that security
+requirements are implemented from the beginning. And, these requirements are no
+longer defined by spreadsheets and quickly-dated documents, but where possible
+they are defined by code. This code will make crystal clear, to anyone who would
```

Comment:
```suggestion
they are defined by code and continuously verified. This code will make crystal clear, to anyone who would
```

I think the continuous verification is super important, as we're not just checking the boxes once to please an auditor, but building verifiable security into the software delivery system.